storybook/storybook-react/src/Icon/Icon.stories.tsx

```diff
@@ -13,7 +13,7 @@ const meta = {
   argTypes: {
     size: {
       control: { type: 'select' },
-      options: ['level-3', 'level-4', 'level-5', 'level-6', 'level-7'],
+      options: ['level-3', 'level-4', 'level-5', 'level-6'],
```

Comment:
Icon doesn't align to the text anymore in the [Link component](https://amsterdam.github.io/design-system/demo-DES-534-update-line-heights-and-text-levels/?path=/docs/react_navigation-link--docs) amd the [Pagination component](https://amsterdam.github.io/design-system/demo-DES-534-update-line-heights-and-text-levels/?path=/docs/react_navigation-pagination--docs)

Reply:
Ah, that bug must have been introduced with the list items going to the regular level. Fixed now.

Reply:
It's fixed for Link, Pagination is still off I think

Reply:
Ah. Done as well.